department-of-veterans-affairs/caseflow
# Remove Prometheus service since we will be using Datadog

### Description
Since we are actively using Datadog we decided to remove Prometheus. 


### Testing Plan
1. Deploy to UAT
2. Ensure the site is up and running
3. Check DD healthy host count for UAT servers https://app.datadoghq.com/dashboard/pww-yy2-j9d/load-balancer-healthy-host-counts?from_ts=1568725645197&fullscreen_widget=262865873&live=true&tile_size=m&to_ts=1568729245197&fullscreen_section=overview
4. Check Kibana logs for prometheus requests receiving route not found
